renovate.json

```diff
@@ -15,7 +15,7 @@
         "/^assets/config/config\\.yaml$/"
       ],
       "matchStrings": [
-        "#\\s*renovate:\\s*datasource=(?<datasource>\\S+)\\s+depName=(?<depName>\\S+)(\\s+packageName=(?<packageName>\\S+))?(\\s+registryUrl=(?<registryUrl>\\S+))?(\\s+versioning=(?<versioning>\\S+))?\\s*\\n\\s*\\S+:\\s*\"?(?<currentValue>[^@\"\\s]+)(?:@sha256:(?<currentDigest>[a-f0-9]+))?\"?"
+        "#\\s*renovate:\\s*datasource=(?<datasource>\\S+)\\s+depName=(?<depName>\\S+)(\\s+packageName=(?<packageName>\\S+))?(\\s+registryUrl=(?<registryUrl>\\S+))?(\\s+versioning=(?<versioning>\\S+))?\\s*\\n\\s*\\S+:\\s*\"?(?<currentValue>[^@\"\\s]+)(?:@(?<currentDigest>sha256:[a-f0-9]+))?\"?"
```

Comment:
nit: we could follow https://github.com/containers/container-libs/blob/b0f86df5a66598b80ddf7d4cbd88165981d3370c/image/docker/reference/reference.go#L20 and use a regex for the algorithm instead of hardcoding to sha256 (although I have personally never seen any other algorithm in use)